Project_1.cpp

```diff
@@ -29,7 +26,6 @@
             colors[i] = getRandomColor();
         }
 
-
         while (window.isOpen())

```

Comment:
You should consider what you put into a while loop. Setting the condition to be while the window is open would cause whatever instructions inside the loop to repeat until the window is closed.

You don't want to keep re-drawing the grid over and over again.